src/main/webapp/lobby.js

```diff
@@ -38,3 +38,5 @@ function createUsernameElement(username) {
     node.appendChild(textnode);
     return node;
   }
+
+document.getElementById("url").innerHTML = "Share the link!:<br>" + (window.location.href);
```

Comment:
Oh interesting, does this display the link in the lobby for other users to share? Could you include a screenshot of this in your PR description?

Reply:
+1 would love to see this.